tests/util/test_unit_conversion.py

```diff
@@ -32,59 +35,31 @@
 )
 
 INVALID_SYMBOL = "bob"
+_CONVERTERS: list[type[BaseUnitConverter]] = [
+    obj
+    for _, obj in inspect.getmembers(unit_conversion)
+    if inspect.isclass(obj)
+    and issubclass(obj, BaseUnitConverter)
+    and obj != BaseUnitConverter
+]
```

Comment:
Another possibility is to make this list explicit, and then add a unit test to ensure that the explicit list contains all converters.

Reply:
Sometimes I wonder if we should use the Hypothesis library in HA for cases like these.